src/components/team-gallery/team-gallery.vue

```diff
@@ -0,0 +1,172 @@
+<template>
+  <section class="grid">
+    <ul class="team-gallery__list">
+      <li
+        v-for="member in team "
+        :key="member.id"
+        class="team-gallery__list-item"
+      >
+        <app-link
+          :to="$localeUrl({ name: 'team-slug', params: { slug: member.slug } })"
+          class="team-gallery-member__link"
+          tabindex="-1"
+        >
+          <dato-image
+            class="team-gallery-member__image"
+            :src="member.image.url"
+            :alt="member.name"
+            :width="member.image.width"
+            :height="member.image.height"
+            loading="lazy"
+            sizes="(min-width: 800px) 20vw, (min-width: 500px) 33vw, 50vw"
+          />
+        </app-link>
+        <div class="team-gallery-member__details">
+          <app-link
+            :to="$localeUrl({ name: 'team-slug', params: { slug: member.slug } })"
+            class="team-gallery-member__link"
+          >
+            <p class="h4">
+              {{ member.name }}
+            </p>
+          </app-link>
+          <span
+            v-if="member.jobTitle"
+            class="body-detail"
+          >{{ member.jobTitle }}</span>
+        </div>
+      </li>
+    </ul>
+  </section>
+</template>
+
+<script setup lang="ts">
+  defineProps<{
+    team: Array<{
+      id: string,
+      slug: string,
+      name: string,
+      lastName: string,
+      jobTitle: string,
+      image: {
+        url: string,
+        width: number,
+        height: number,
+      },
+    }>
+  }>();
+</script>
+
+<style>
+  :root {
+    --team-gallery-grid-columns: 1;
+  }
+
+  .team-gallery__list {
+    display: grid;
+    grid-template-columns: repeat(var(--team-gallery-grid-columns), 1fr);
+    gap: 16px;
+  }
+
+  .team-gallery-member__link {
+    display: block;
+    height: auto;
+    overflow: hidden;
+  }
+
+  .team-gallery-member__link:hover .team-gallery-member__image {
+    transform: scale(1.05);
+  }
+
+  .team-gallery__list-item {
+    position: relative;
+  }
+
+  .team-gallery-member__image {
+    display: block;
+    height: 100%;
+    object-fit: cover;
+    transition: transform 200ms ease-in-out;
+  }
+
+  .team-gallery-member__details {
+    position: absolute;
+    padding: 1rem;
+    z-index: 1;
+    bottom: 0;
+    width: 100%;
+    background-color: var(--brand-yellow);
+  }
+
+  @media (min-width: 768px) {
+    .team-gallery-member__details {
+      opacity: 0;
```

Comment:
Do we actually need the hover animation? I do think it looks fancy, but I'm wondering if it doesn't make the page less accessible, since you can't scan the page easily anymore. You have to hover each item if you're looking for someone.

Reply:
After some brainstorming sesh, we redesigned the component so no hover state is needed now user can scan through the team gallery with ease. 👍🏻